ci/check-style.sh

```diff
@@ -40,18 +40,18 @@ find google/cloud -name '*.h' -print0 |
   xargs -0 awk -f "${BINDIR}/check-include-guards.gawk"
 
 replace_original_if_changed() {
-  if [[ $# != 1 ]]; then
+  if [[ $# != 2 ]]; then
     return 1
   fi
 
-  local file="$1"
-  shift
+  local original="$1"
+  local reformatted="$2"
 
-  chmod --reference="${file}" "${file}.tmp"
-  if cmp -s "${file}" "${file}.tmp"; then
-    rm -f "${file}.tmp"
+  chmod --reference="${original}" "${reformatted}"
```

Comment:
I'm not sure why I didn't originally put the "chmod" in the "mv" block, but we can do just that if we get back here.